src/hammer-vlsi/technology/nangate45/nangate45.tech.json

```diff
@@ -0,0 +1,153 @@
+{
+  "name": "Nangate45 Library",
+  "grid_unit": "0.005",
+  "time_unit": "1 ps",
+  "installs": [{
+    "path": "nangate45",
+    "base var": "technology.nangate45.install_dir"
+  }],
+
+  "gds map file": "nangate45/layermap.txt",
+  
+  "drc decks": [{
+    "tool name": "openroad",
+    "deck name": "drc",
+    "path": "nangate45/FreePDK45.lydrc"
+  }],
+  "additional_drc_text": "",
+  
+  "lvs decks": [],
+  "additional_lvs_text": "", 
+
+  "libraries": [{
+    "lef file": "nangate45/lef/NangateOpenCellLibrary.tech.lef",
+    "provides": [{"lib_type": "technology"}]
+  },{
+    "nldm liberty file": "nangate45/lib/NangateOpenCellLibrary_typical.lib",
+    "lef file": "nangate45/lef/NangateOpenCellLibrary.macro.lef",
+    "gds file": "nangate45/gds/NangateOpenCellLibrary.gds",
+    "corner": {
+      "nmos": "typical",
+      "pmos": "typical",
+      "temperature": "25 C"
+    },
+    "supplies": {
+      "VDD": "1.10 V",
+      "GND": "0 V"
+    },
+    "provides": [{
+      "lib_type": "stdcell",
+      "vt": "none"
+    }]
+  }],
+  "stackups": [{
+    "name": "asap7_3Ma_2Mb_2Mc_2Md",
```

Comment:
Name and layers should be checked.

Reply:
Presumably this will happen when we validate P&R

Reply:
i ran the dump_stackup_to_json.tcl in innovus on the tech-lef, so they should be close to correct

the stackup name is arbitrary. there is no official naming convention for openroad's nangate45

Reply:
ASAP7's stackup was also not named. I made that stackup name based on how many metal layers of each type there were. You could follow a similar convention.

Reply:
i did copy harrison's asap7 naming convention

Reply:
I think we just need `s/asap7/nangate45` on this line.